build_tools/github/build_minimal_windows_image.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+set -e
+set -x
+
+PYTHON_VERSION=$1
+BITNESS=$2
+
+if [[ "$PYTHON_VERSION" == "36" || "$BITNESS" == "32" ]]; then
+    # Python 3.6 and 32-bit architectures are not supported
+    # by the official Docker images: Tests will just be run
+    # on the host (instead of the minimal Docker container).
+    exit 0
+fi
+
+if [[ "$PYTHON_VERSION" == "37" ]]; then
+    WHEEL_NAME=cp"$PYTHON_VERSION"m-win_amd64.whl
+else
+    # Different name for Python 3.8 and newer
+    WHEEL_NAME=cp"$PYTHON_VERSION"-win_amd64.whl
+fi
+
+WHEEL_NAME=scikit_learn-$SCIKIT_LEARN_VERSION-cp$PYTHON_VERSION-$WHEEL_NAME
+
+# Find the repaired wheel because there is no a way
+# to access to the path in a straightforward manner
+
+# The repaired wheel is under a temporary directory that may
+# remove folders concurrently with respect to "find" command
+TEMP_FOLDER="$HOME/AppData/Local/Temp"
+WHEEL_PATH=$(find $TEMP_FOLDER -type d -name "repaired_wheel" 2>/dev/null); \
+           echo "Ignoring concurrently deleted temporary folders."
```

Comment:
Maybe the following can work in a simpler and more stable manner?

```suggestion
WHEEL_PATH=$(ls $TEMP_FOLDER/*/repaired_wheel)
```

Reply:
Good solution. I think that this should work and reduce the diff.